src/es/manhwases/build.gradle

```
@@ -0,0 +1,10 @@
+ext {
+    extName = 'Manhwas.es'
+    extClass = '.ManhwasEs'
+    themePkg = 'madara'
+    baseUrl = 'https://www.manhwas.es'
+    overrideVersionCode = 1
```

```suggestion
    overrideVersionCode = 0
```
new multisrc extensions can start from 0